api/keto/acl/v1/watch_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package keto.acl.v1;
+
+import "keto/acl/v1/acl.proto";
+import "google/api/client.proto";
+
+option go_package = "github.com/ory/keto/api/keto/acl/v1;acl";
+option csharp_namespace = "Ory.Keto.Acl.V1";
+option java_multiple_files = true;
+option java_outer_classname = "WatchServiceProto";
+option java_package = "sh.ory.keto.acl.v1";
+option php_namespace = "Ory\\Keto\\Acl\\V1";
+
+// The service to watch for changes in the system,
+// such as for Access Control Lists and namespace configs.
+service WatchService {
```

Comment:
The watch service will require a lot of background setup first, lets add the API definition together with that functionality.

Reply:
I'll remove it.